React/Views/RCTView.m

```diff
@@ -456,6 +606,11 @@ - (BOOL)accessibilityPerformPress
   } else if (_onAccessibilityTap) {
     _onAccessibilityTap(nil);
     return YES;
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+  } else if (_onClick) {
+    _onClick(nil);
+    return YES;
+#endif // ]TODO(macOS ISS#2323203)
```

Comment:
Do you imagine that at some point we might be able to upstream things like this and just have it not do anything on platforms that don’t have the concept of clicks?

Reply:
This is a weird one.  On iOS returning NO from `accessibilityActivate` can make the system simulate a touch on the view instead which ends up indirectly causing an `onPress` on a touchable get called.   On macOS this doesn't happen.   There are a number of cases where a View or Touchable does *not* have an `onAccessiblityAction` handler but does have on `onPress` and expects a VoiceOver double tap to call it.   It's an important case to support and this was the most obvious way to simulate a click -- we do it in other places such as `performKeyEquivalent:`.

Reply:
Good to know, thanks for the update! (I’ll actually unresolve this thread, as it’s just good general info for posterity.)

Reply:
BTW, I think I found how mac VoiceOver can simulate a click on VO activate: if the accessibilityActivationPoint prop is set https://developer.apple.com/documentation/appkit/nsaccessibility/1535149-accessibilityactivationpoint?language=objc.   Might be worth further investigation in the future, but don't want to add risk to this change by introducing another property.